README.md

```diff
@@ -27,9 +27,14 @@ git clone https://github.com/etesync/server-skeleton.git
 cd server-skeleton
 
 # Set up the environment and deps
-virtualenv -p python3 venv  # If doesn't work, try: virtualenv3 venv
+virtualenv -p python3 venv # If doesn't work, try: virtualenv3 venv
 source venv/bin/activate
 
+# if the commands on top don't work (like Debian) do :
+python3 -m venv .
```

Comment:
Maybe include it as an alternative in line 30 like `virtualenv3 venv` already is?
Additionally, if you change this to `python3 -m venv venv` all of the commands (including the `source` one) will be consistent across distros/variations.